.github/workflows/release.yaml

```diff
@@ -79,18 +79,23 @@ jobs:
           fork-user: prql-bot
 
   build-deb-package:
-    # TODO: currently this doesn't publish it as an artifact; would be a welcome contribution
-    # to add that (and eventually as a release asset)
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+          - os: ubuntu-latest
+            target: aarch64-unknown-linux-musl
+    needs: build-prqlc
+    permissions:
+      contents: write
     steps:
-      - name: 📂 Checkout code
-        uses: actions/checkout@v4
-      - name: 👷 Build prql-compiler
-        run: cargo build --release -p prqlc
       - name: Copy files into .deb package
         run: |
           mkdir -p .debpkg/usr/bin
-          cp target/release/prqlc .debpkg/usr/bin/prqlc
+          cp target/*/release/prqlc .debpkg/usr/bin/prqlc
```

Comment:
I think we need to use `actions/download-artifact` to download the artifact.
https://github.com/PRQL/prql/blob/8ed38fbef546289c846a2820e79356f8ac91a318/.github/actions/build-prqlc/action.yaml#L89-L97

Reply:
So I tried that now, but I am struggling with this. Any more ideas? Wanna give it a try? The PR is open for edits by maintainers.

Reply:
Look for other repositories doing similar things. For example I know bendsql does something similar with this job.
https://github.com/datafuselabs/bendsql/blob/b9033d327ab21e8fb18ead2613d848cabb3b53a3/.github/workflows/release.yml#L97-L124